### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by kcheverie**
added a footer!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,5 +20,10 @@ <h1>App!</h1>
     <img src="https://images.unsplash.com/photo-1517331156700-3c241d2b4d83?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1768&q=80" alt="a basket of orange and calico kittens in front of a lilac tree">
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris elit nisl, congue eu varius a, tristique sit amet dolor. Aenean lacinia tortor est, eget congue justo mattis et. Curabitur vitae nibh vitae arcu faucibus lacinia. Cras tempus eleifend dolor et pretium. Morbi hendrerit gravida odio eget laoreet. Donec laoreet metus metus. Suspendisse pharetra pulvinar dictum. Nulla viverra luctus velit, vel elementum nibh consequat sed. Mauris molestie volutpat lorem, non hendrerit lacus tempus eget. Duis ultricies elit at lacus pharetra faucibus. Nulla ultrices lacus sed sodales ullamcorper. Ut sollicitudin dui id sem imperdiet, at tempus felis auctor.</p>
   </main>
+  <div>
```
Comment (EstherEdell) [inline index.html]: please change this to a <footer> element - it's more semantic!

**Review by EstherEdell — CHANGES_REQUESTED** [review]
one small issue

File: index.html (hunk 1 above)
Comment (kcheverie) [reply index.html]: good catch! i'll fix that

**Review by EstherEdell — APPROVED** [review]
looks great! approved!

**Pull request merged**